### DISCUSSION TITLE
Error updating repository

### DISCUSSION THREAD

**Discussion by clriis** [abraunegg/onedrive#2000] [Q&A] [answered]
Hi,

Onedrive has been working flawlessly on my headless Pi4-64bit system for long, but now I get these errors when updating repositories.

![image](https://user-images.githubusercontent.com/16999072/172767745-5e012d2a-69fc-4d95-a5da-840c0bbe704c.png)

**Comment by abraunegg** [comment, marked as answer]
@clriis 
This will be because for some reason your system does not have the correct digital signatures ...

Unfortunately this is nothing I can assist with.

Suggest you review all the steps listed [here](https://github.com/abraunegg/onedrive/blob/master/docs/ubuntu-package-install.md) and re-do them to see if that helps, otherwise seek support from Debian / Raspbian as this is an Operating System issue - not an issue with this client.

**Comment by boblemus** [comment]
I had the same problem. I run Ubuntu 20.04 LTS.
All I had to do was to update the repository key:
wget -qO - https://download.opensuse.org/repositories/home:/npreining:/debian-ubuntu-onedrive/xUbuntu_20.04/Release.key | sudo apt-key add -
Then apt-get update and the error had disappeared.
So as @abraunegg mentioned, try to find the instructions for how to update the key for your system, and you should be good to go.

**Reply by clriis** [reply]
Thanks alot

wget -qO - https://download.opensuse.org/repositories/home:/npreining:/debian-ubuntu-onedrive/Debian_11/Release.key | sudo apt-key add -

Did the trick.
